core/controller/src/main/scala/whisk/core/controller/WebActions.scala

```diff
@@ -118,7 +125,7 @@ private case class Context(
 
     def toActionArgument(user: Option[Identity], boxQueryAndBody: Boolean): Map[String, JsValue] = {
         val queryParams = if (boxQueryAndBody) {
-            Map(propertyMap.query -> JsString(query.render(new StringRendering, StandardCharsets.UTF_8).get))
+            Map(propertyMap.query -> JsString(query.toString))
```

Comment:
can you look at the toString implementation? can we bottom out into where it ends up?